### ISSUE TITLE
Support a VSCode Extension or helper application in the future

### ISSUE THREAD

**Issue by nickshanks347** [getcursor/cursor#1245]
**Is your feature request related to a problem? Please describe.**
This is not related to a problem

**Describe the solution you'd like**
VSCode feels like home, cursor feels like an editor on steroids. The one thing that's stopping me buying a subscription is that it isn't supported in native VSCode. I understand it needs access to parts of the IDE that probably can only be accessed/manipulated by writing your own IDE but there must be a way to have an extension in VSCode that can communicate with a helper application.

This would be really great to see in the future, because I just don't think I can move away from VSCode.

**Comment by avlm** [comment]
agree, I use vscode with github login to keep my extensions and apps always up to date between the computers I use to work. An official cursor vscode extension would be great!
